test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -225,41 +225,37 @@ public static void waitFor(Supplier<Boolean> supplier) throws InterruptedExcepti
      *
      * @param pid parent process identifier
      * @return list of children process identifiers
-     *
-     * @throws IOException
+     * @throws IOException {@inheritDoc}
+     * @throws InterruptedException {@inheritDoc}
      */
-    private static List<Long> getChildPIDs (long pid) {
+    private static List<Long> getChildPIDs (long pid) throws IOException, InterruptedException {
```

Comment:
The method now throws these exceptions to the caller. I do not the caller of this method in this file meaning its probably being called outside of this file. Did you make sure all the callers are handling this exception and logging correctly, as a cautionary measure?

Reply:
I think there isn't any semantic changes from this change, since in the previous implementation the catch clause was just re-throwing the exceptions (concealed by the return in finally so we never got them). But yeah I will check the callers to make sure it, thanks!